src/mlpack/methods/lsh/lsh_search_impl.hpp

```diff
@@ -270,7 +270,7 @@ void LSHSearch<SortPolicy, MatType>::Train(MatType referenceSet,
     // and the corresponding offset be 'offset_i'.  Then the key of a single
     // point is obtained as:
     // key = { floor((<proj_i, point> + offset_i) / 'hashWidth') forall i }
-    arma::mat offsetMat = arma::repmat(offsets.unsafe_col(i), 1,
+    arma::mat offsetMat = repmat(offsets.unsafe_col(i), 1,
                                        this->referenceSet.n_cols);
```

Comment:
```suggestion
                                 this->referenceSet.n_cols);
```
No worries, we should at least align the second line though. :+1: